sql/2021/performance/lcp_element.sql

```diff
@@ -0,0 +1,19 @@
+#standardSQL
+# LCP element node
+# This is a simplified query - the lcp_element_data.sql one will probably be used instead. Leaving this here for reference for now.
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  JSON_EXTRACT_SCALAR(payload, "$._performance.lcp_elem_stats[0].nodeName") AS lcp_node,
+  COUNT(DISTINCT url) AS pages,
+  SUM(COUNT(DISTINCT url)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
+  COUNT(DISTINCT url) / SUM(COUNT(DISTINCT url)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct
+FROM
+  --`httparchive.pages.2021_07_01_*`
+  `httparchive.sample_data.pages_*`
```

Comment:
Can you go through all the queries to make sure they're referencing the full dataset?
```suggestion
  `httparchive.pages.2021_07_01_*`
```